recipes-kernel/linux/linux-intel-acrn_5.4.inc

```diff
@@ -11,7 +11,7 @@ KMETA_BRANCH = "yocto-5.4"
 
 LINUX_VERSION ?= "5.4.46"
 SRCREV_machine ?= "3c038f1968310a1d08245b5f1a550e4e0e06d25d"
-SRCREV_meta ?= "b8c82ba37370e4698ff0c42f3e54b8b4f2fb4ac0"
+SRCREV_meta ?= "627191aa87d971e153f95beac4d9e45aea0e9b65"
```

Comment:
Thank you for updating SRCREV_meta. It is important to watch for changes to the yocto-kernel-cache when we update the kernel to a newer dot release.